### ISSUE TITLE
Add 'Recurring Event' badge to distinguish recurring events

### ISSUE THREAD

**Issue by natebyerley** [az-digital/az_quickstart#3139]
This is a low priority feature request, and is worthy of discussion.  I would bundle this request in with other feature changes around events/calendar widget in the future. 

## Motivation
If website visitor uses calendar widget to filter for events, they are unable to see if an event is recurring - recurring and one-time events appear identical.  

![Screenshot 2024-01-31 at 11 58 16 AM](https://github.com/az-digital/az_quickstart/assets/97116860/81d61044-347b-41ce-964c-5332e58242a6)

### Is your feature request related to a problem? Please describe.
It may be helpful for website visitors to know that an event that appears only once in a view is actually a recurring event.

## Proposed Resolution
Rather than list all recurring dates as an indicator of recurring status, client idea was to add a badge to indicate that an event was recurring.

### Describe the solution you'd like

![Screenshot 2024-01-31 at 11 58 16 AM](https://github.com/az-digital/az_quickstart/assets/97116860/5feccacd-5c1a-4cd0-8084-664a5eb37025)

### Describe alternatives you've considered
Advise clients to use individual events for event 'series' rather than recurring events.

**Comment by bberndt-uaz** [comment]
This is definitely an interesting idea to consider, but personally I think I would prefer to use additional text to present recurrence information in a similar way to Google Calendar, Outlook, or other calendar apps. See https://github.com/az-digital/az_quickstart/issues/2028#issuecomment-1329854432.
